server/src/logger/devLogger.ts

```diff
@@ -0,0 +1,33 @@
+import { format, createLogger, transports } from 'Winston';
```

Comment:
I needed this to be lower-case to get the code to compile

Reply:
Done!